paseos/tests/attitude_test.py

```diff
@@ -1,12 +1,96 @@
 """Tests to see whether the attitude and disturbance models work as intended"""
-
 import numpy as np
 import pykep as pk
 import sys
 
 sys.path.append("../..")
 import paseos
-from paseos import ActorBuilder, SpacecraftActor
+from paseos import ActorBuilder, SpacecraftActor, load_default_cfg
+
+
+def gravity_disturbance_cube_test():
```

Comment:
The torque also depends on the attitude of the satellite itself. Are you considering a case where the satellite at time 0 is in equilibrium (i.e., gravity forces are directed for both the "faces" of the satellite toward the Earth's centre of mass)?

Reply:
I've included some more description in accordance with what was discussed